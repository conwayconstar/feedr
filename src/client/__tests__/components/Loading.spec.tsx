import { shallow } from 'enzyme';
import React from 'react';
import Loading from '../../components/Loading';

describe('Loading component', () => {
  const wrapper = shallow(<Loading />);

  it('Loading should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
