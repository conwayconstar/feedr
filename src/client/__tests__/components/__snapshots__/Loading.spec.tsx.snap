// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component Loading should match snapshot 1`] = `
<div
  className="d-flex justify-content-center"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
